ui/app/models/action-instance.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
```

Comment:
A running action; has a `socket` property that we do a bunch of event handling on in our nomad-actions service.

This is a relatively rare model without a server-side equivalent, and only exists to give us a nice reproducible way of handling running actions.